priv/repo/seeds.exs

```diff
@@ -9,3 +10,39 @@ CoursePlanner.User.changeset(%CoursePlanner.User{},
     role: "Coordinator"},
     :seed)
 |> CoursePlanner.Repo.insert!
+
+CoursePlanner.SystemVariable.changeset(%CoursePlanner.SystemVariable{},
+  %{
+    key: "Program name",
```

Comment:
I would suggest the keys to have a more strict letter case, like screaming snake case `PROGRAM_NAME`